test/fixtures/patterns/components/button/yet-another.html

```diff
@@ -1,5 +1,6 @@
 ---
 name: Something Yet Again
+order: -1
 ---
 
-Hmmm, what goes here.
+Hmmm, what goes here.
```

Comment:
I still don't fully understand why GitHub yells at us when there isn't a newline at the end of the file, but this red arrow makes me sad...

<img width="297" alt="screen shot 2018-07-02 at 2 07 09 pm" src="https://user-images.githubusercontent.com/69633/42186359-3cc100d4-7e01-11e8-95a3-27c1071e08a2.png">

Reply:
✔️